.github/workflows/CI.yml

```diff
@@ -119,6 +119,34 @@ jobs:
       run: |
         make test-python
 
+  arkouda_chpl_portability:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        chpl-version: ['1.30.0']
+    container:
+      image: chapel/chapel:${{matrix.chpl-version}}
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install dependencies
+      run: |
+        apt-get update && apt-get install -y -V ca-certificates lsb-release wget
+        wget https://apache.jfrog.io/artifactory/arrow/$(lsb_release --id --short | tr 'A-Z' 'a-z')/apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb
+        apt-get install -y -V ./apache-arrow-apt-source-latest-$(lsb_release --codename --short).deb
+        apt-get update && apt-get install -y libhdf5-dev hdf5-tools libzmq3-dev python3-pip libarrow-dev libparquet-dev libcurl4-openssl-dev libidn2-dev
+        make install-iconv
+        echo "\$(eval \$(call add-path,/usr/lib/x86_64-linux-gnu/hdf5/serial/))" >> Makefile.paths
+    - name: Check chpl version
+      run: |
+        chpl --version
+    - name: Build/Install Arkouda
+      run: |
+        make
+        python3 -m pip install -e .[dev]
+    - name: Arkouda unit tests
+      run: |
+        make test-python
+
```

Comment:
I don't think we need a duplicate of the `arkouda_chpl_portability`. You should be able to just update the `chapel-version` matrix to `["1.29.0", "1.30.0"]`. This was we aren't duplicating things unnecessarily.

Reply:
i think the reason ben did this was because we decided to do "recommend 1.31" and "drop 1.29" as separate PRs. They remove the the 1.29 version in https://github.com/Bears-R-Us/arkouda/pull/2551

Reply:
sorry i completely misunderstood what u were saying, this is why i shouldn't comment on PRs right after waking up 😅

Reply:
Ya, like Pierce mentioned, I was doing this because I was dropping 1.29 in another PR, so this way there are no merge conflicts. Seeing https://github.com/Bears-R-Us/arkouda/pull/2551#issuecomment-1625580404, I'll update this one though.